docs/proposals/custom-wbt-settings.md

```diff
@@ -0,0 +1,50 @@
+# Proposal: Support option to change WBT settings
+
+Last updated: 07.12.2021
+
+
+## Abstract
+
+CSI should have an opportunity to automate WBT (writeback throttling) disabling for specific Volumes.
+
+## Background
+
+Some Storage Systems, which use Baremetal CSI, may have performance issues due to incorrect WBT settings.
```

Comment:
should we mention that there is a [bug ](https://access.redhat.com/solutions/4904681) in WBT in some versions of kernel which has a recommended workaround in disabling this feature?  just a thought.

Reply:
added, thank you